### PR TITLE
[kong] release 2.1.0 to next

### PR DESCRIPTION
**Pull request by rainest**
#### What this PR does / why we need it:
Releases 2.1.0.

#### Special notes for your reviewer:
Contingent on https://github.com/Kong/charts/pull/349. Do not merge until that is complete.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] PR is based off the current tip of the `next` branch and targets `next`, not `main`
- [x] Title of the PR and commit headers start with chart name (e.g. `[kong]`)

### CODE REVIEW

File: charts/kong/UPGRADE.md [hunk 1]
```diff
@@ -57,6 +58,19 @@ text ending with `field is immutable`. This is typically due to a bug with the
 If you encounter this error, deleting any existing `init-migrations` jobs will
 clear it.
 
+## 2.1.0
+
+### Migration off Bintray
```
Comment (mflendrich) [inline charts/kong/UPGRADE.md:63]: This happened in 2.0.0, didn't it?

I think that adding the remark about this change to 2.1.0 (while migration of bintray happened in the previous minor release) may be a source of confusion.

Comment (rainest) [reply charts/kong/UPGRADE.md:63]: Partially. We didn't get everything in the last release, and the 2.1 defaults are the first set that has the new repos for all images.

File: charts/kong/UPGRADE.md [hunk 2]
```diff
@@ -57,6 +58,19 @@ text ending with `field is immutable`. This is typically due to a bug with the
...
+
+Bintray, the Docker registry previously used for several images used by this
+chart, is [sunsetting May 1,
+2021](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/).
+
+The chart default values.yaml now uses the new Docker Hub repositories for all
+affected images. You should check your release values.yamls to confirm that
+they do not still reference Bintray repositories. If they do, update them to
+use the Docker Hub repositories now in the default values.yaml.
```
Comment (mflendrich) [inline charts/kong/UPGRADE.md]:
```suggestion
The chart default `values.yaml` now uses the new Docker Hub repositories for all
affected images. You should check your release `values.yaml` files to confirm that
they do not still reference Bintray repositories. If they do, update them to
use the Docker Hub repositories now in the default `values.yaml`.
```

**Review by mflendrich — APPROVED** [review]
one small editorial comment, otherwise LGTM

**Pull request merged**